resources/quality/malyan_m200/malyan_m200_0.175.inst.cfg

```diff
@@ -0,0 +1,23 @@
+[general]
+version = 2
+name = M4 Quality
+definition = malyan_m200
+
+[metadata]
+type = quality
+weight = -1
+quality_type = fast
+global_quality = true
```

Comment:
Some files have `global_quality = true` and some don't. I think all `global_quality = true` can be removed.

Reply:
From MachineManager.py, global quality should only be set to true for the default quality profile on the machine.  This is the default quality profile, hence it being the only one set to global_quality=true
https://github.com/Ultimaker/Cura/blob/master/cura/Settings/MachineManager.py#L981